tests/operators/test_sql.py

```diff
@@ -385,6 +387,82 @@ def test_fail_min_sql_max_value(self, mock_get_db_hook):
             operator.execute()
 
 
+class TestColumnCheckOperator(unittest.TestCase):
```

Comment:
IMO this can be a `pytest` test rather than continuing to use `unittest`.

Reply:
Is there a reason to use one over the other? I was just copying the test pattern from the other ones.

Reply:
We prefer pytest test type unless there are good reasons not to (for new test)